### PR TITLE
Updated translation of chapter 7 in Romanian 🇷🇴

### PR DESCRIPTION
**Pull request by Adin3**
Am modificat cateva greseli gramaticale si lipsa unor diacritice

### CODE REVIEW

**Review by alexandruradovici — APPROVED** [review]
Te rog trimite-l spre repository-ul original si pune un link aici.

**Comment by Adin3** [comment]
am dat pull request: https://github.com/richardanaya/tour_of_rust/pull/492

**Comment by alexandruradovici** [comment]
Care este numele tau intreg?

**Comment by Adin3** [comment]
Neacșu Adrian

**Comment by alexandruradovici** [comment]
Te rog trimite pull request-ul in upstream si pune un link aici.

File: lessons/ro/chapter_7.yaml [hunk 1]
```diff
@@ -31,7 +31,7 @@
     * Structurile nu pot moșteni funcții de la o structura părinte.
```
Comment (ScourgeNest) [inline lessons/ro/chapter_7.yaml]: structură*

File: lessons/ro/chapter_7.yaml [hunk 2]
```diff
@@ -31,7 +31,7 @@
...
 
     Acestea fiind spuse, Rust implementează o mulțime de caracteristici ale unui limbaj de programare,
-    deci probabail nu vă vor deranja aceste lipsuri.
+    deci probabil nu vă vor deranja aceste lipsuri.
 - title: Încapsulare prin matode
```
Comment (ScourgeNest) [inline lessons/ro/chapter_7.yaml]: metode*

**Pull request closed without merging**